server/index.ts

```diff
@@ -36,4 +38,6 @@ nextjs.nextApp.prepare().then(async () => {
   app.listen(port, () => {
     console.log(`\n\nstarted on port ${port}\n\n`); // tslint:disable-line no-console
   });
+
+  app.use('/api-docs', swaggerUi.serve, swaggerUi.setup(swaggerDocument));
```

Comment:
I think that we should wrap this in `if nodeenv === prod`, @timmyichen @allella do we want API docs to be public in the MVP

Reply:
Also maybe change it to `/api/v1/(docs||speccy)` to represent that these are the docs for the `API v1`